lib/metadata/virtual_container_host.go

```diff
@@ -96,6 +100,26 @@ type VirtualContainerHostConfigSpec struct {
 
 	// Allow custom naming convention for containerVMs
 	ContainerNameConvention string
+
```

Comment:
@hickeng I did a rebase against master after your PR went in and I'm seeing some things showing up now as green in the diff which leads me to think you removed some things and I've unintentionally re-added them -- can you confirm that e.g. this code segment was stripped by your change?